AstronomyTests/MarsRoverClientTests.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  MarsRoverClientTests.swift
+//  MarsRoverClientTests
+//
+//  Created by Hunter Oppel on 5/18/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import XCTest
+@testable import Astronomy
+
+/*
+ Does decoding work?
+ Does decoding fail when given bad data?
+ Does it build the correct URL?
+ Does it build the correct URLRequest?
+ Are the search results saved properly?
+ Is the completion handler called when data is good?
+ Is the completion handler called when data is bad?
+ Is the completion handler called when networking fails?
+ */
+
+class MarsRoverClientTests: XCTestCase {
+    
+    private let fakeRover = MarsRover(name: "Friend",
+                                      launchDate: Date(),
+                                      landingDate: Date() + 2,
+                                      status: .active,
+                                      maxSol: 1,
+                                      maxDate: Date(),
+                                      numberOfPhotos: 16,
+                                      solDescriptions: [])
+    
+    func testForPhotosWithServer() {
+        let roverExpectation = expectation(description: "Wait for rover info")
+        
+        let controller = MarsRoverClient(networkLoader: URLSession(configuration: .ephemeral))
+        var roverInfo: MarsRover?
```

Comment:
Perhaps make this explicitly unwrapped:
```suggestion
        var roverInfo: MarsRover!
```